tsp-cli-client

```diff
@@ -310,17 +317,66 @@ if __name__ == "__main__":
                 print("Trace UUID is missing")
                 sys.exit(1)
 
+        if (options.get_xy_tree):
+            if options.uuid is not None:
+
+                output_descriptor = get_descriptor(
+                    options.uuid, options.get_xy_tree)
+                if (output_descriptor is None):
+                    sys.exit(1)
+
+                response = tsp_client.fetch_xy_tree(
+                    options.uuid, options.get_xy_tree)
+                if response.status_code == 200:
+                    tree = response.model.model
+
+                    if (output_descriptor.type == "DATA_TREE"):
+                        treeModel = TreeModel(tree.entries, tree.descriptors)
+                    else:
+                        treeModel = TreeModel(tree.entries)
+                    treeModel.print()
+                    sys.exit(0)
+                else:
+                    sys.exit(1)
+            else:
+                print("Trace UUID is missing")
+                sys.exit(1)
+
+        if (options.get_xy):
+            if options.items is None or options.times is None:
+                print("Provide requested --items and --times for the XY data")
+                sys.exit(1)
+
+            if options.uuid is not None:
+                output_descriptor = get_descriptor(
+                    options.uuid, options.get_xy)
+                if (output_descriptor is None):
+                    sys.exit(1)
+
+                parameters = {TspClient.REQUESTED_TIME_KEY: list(map(int, options.times)),
```

Comment:
Try checking before if the options.times/items is not an empty list. or else we get 500 errors.

Reply:
> Try checking before if the options.times/items is not an empty list. or else we get 500 errors.

Good catch; thanks. Done (pushed).

Reply:
How do I pass it on the command-line. I have "--times=1487359244565151798,1487359244701115699" which causes the error below. Any idea?

Traceback (most recent call last):
  File "./tsp-cli-client", line 362, in <module>
    parameters = {TspClient.REQUESTED_TIME_KEY: list(map(int, options.times)),
ValueError: invalid literal for int() with base 10: '1487359244565151798,1487359244701115699'

Reply:
> How do I pass it on the command-line. I have "--times=1487359244565151798,1487359244701115699" which causes the error below. Any idea?
> 
> Traceback (most recent call last):
> File "./tsp-cli-client", line 362, in
> parameters = {TspClient.REQUESTED_TIME_KEY: list(map(int, options.times)),
> ValueError: invalid literal for int() with base 10: '1487359244565151798,1487359244701115699'

Using a space-separated list of long integers, rather than with `=` and `,` as above; e.g.:

`--times 1332170682440133097 1332170682440133098`

Same for `--items`. Both of these options follow how `--paths` and `--uuids` were implemented prior. All of these already match what `-h` (`--help`) and README are proposing currently.